kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/foodproducers/Application.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.good.patterns.challenges.foodproducers;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+public class Application {
+
+    public static void main(String[] args) {
+
+        ExtraFoodShopProducer producerExtraFoodShopProducer = new ExtraFoodShopProducer("ExtraFoodShopProducer",
```

Comment:
The main purpose of this task was to enable adding (as easily as possible) new producers and their products. Now I have to add producer to the main method, and also add products below - in which case I have to remember to provide correct name of the shop. Think how to make it simpler and less error-prone.
Ideally it would be, if I have following method somewhere:
addNewProducer(producer, products)